overlord/fdestate/fdemgr.go

```diff
@@ -128,6 +129,17 @@ func Manager(st *state.State, runner *state.TaskRunner) (*FDEManager, error) {
 	runner.AddHandler("add-recovery-keys", m.doAddRecoveryKeys, nil)
 	runner.AddHandler("remove-keys", m.doRemoveKeys, nil)
 	runner.AddHandler("rename-keys", m.doRenameKeys, nil)
+	runner.AddBlocked(func(t *state.Task, running []*state.Task) bool {
+		if strings.HasPrefix(t.Change().Kind(), "fde-") || t.Has("keyslots") {
```

Comment:
is the change kind thing an optimization, I wouldn't do it, it seems fragile, one point of tasks is to be able to mix them, what we can do as I proposed is to add a fde- prefix to the task names instead

Reply:
not intended as an optimization but to match `checkFDEChangeConflict` change prefix check suggested here https://github.com/canonical/snapd/pull/15588#discussion_r2166358610, I can switch both the `AddBlocked` and `checkFDEChangeConflict` to a task level check instead and add `fde-` prefix to task names.

@bboozzoo @pedronis, would this mean also pre-fixing dbx tasks with `fde-` or leave it as it?